src/compute/compute.py

```diff
@@ -68,7 +68,7 @@
 # max content length for upload in bytes
 app.config['MAX_CONTENT_LENGTH'] = int(MAX_FILE_SIZE) * 1024 * 1024
 
-debug = os.environ.get("F7T_DEBUG_MODE", None)
+debug = os.environ.get("F7T_DEBUG_MODE", None) == "True"
```

Comment:
```suggestion
debug = os.environ.get("F7T_DEBUG_MODE", False) == "True"
```
Should it change for consistency?